src/server/api/controllers/favorite.vessels.controller.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+const HttpError = require("../lib/utils/http-error");
+const knex = require("../../config/db");
+
+//POST
+const createFavoriteVessles = async ({ body }) => {
```

Comment:
There is a typo in Vessels and furthermore it should be in singular. So rename to `createFavoriteVessel`.

Reply:
Done and also I corrected the dictation ;)